src/components/card/BasicCard.jsx

```diff
@@ -1,35 +1,40 @@
-export const BasicCard = () => {
+import { RatingPalleteIcon } from "../index";
+
+export const BasicCard = ({ item , children}) => {
+  const {
+    item_name,
+    item_by,
+    item_desc,
+    item_original_price,
+    item_price,
+    item_rating,
+  } = item;
   return (
-    <div class="card w-30 basic">
-      <button class="icon favourite-icon">
-        <i class="fa-regular fa-heart fa-2x"></i>
+    <div className="card w-30 basic">
+      <button className="icon favourite-icon">
+        <i className="fa-regular fa-heart fa-2x"></i>
       </button>
-      <div class="content">
+      <div className="content">
         <img
-          class="card-img"
+          className="card-img"
           src="https://render.fineartamerica.com/images/images-new-artwork/images-medium-5/la-porta-rossa-sulla-salita-guido-borelli.jpg?v=2"
           alt=""
         />
```

Comment:
Alt tag😂

Reply:
please look into it again it has alt tag @Ankur9669